source/ssb-chart/templates/deployment.yaml

```
@@ -90,16 +90,14 @@ spec:
         {{- if .Values.sidecars }}
 {{ toYaml .Values.sidecars | trim | indent 8 | replace "'" "" }}
         {{- end }}
-        {{- if and ( or (eq (toString .Values.cloudsql.enabled) "true") (eq (toString .Values.cloudsql.enabled) "True")) .Values.cloudsql.enabled }}
+        # If CloudSQL enabled is true/True AND useSqlProxy is true/True then create a cloudsql-proxy sidecar
+        {{- if and ( and ( or (eq (toString .Values.cloudsql.enabled) "true") (eq (toString .Values.cloudsql.enabled) "True")) .Values.cloudsql.enabled ( or (eq (toString .Values.cloudsql.useSqlProxy) "true") (eq (toString .Values.cloudsql.useSqlProxy) "True")))  }}
```

This was a little bit hard to read for me. Since this part of the configuration is all about configuring that cloudsql-proxy, couldn't we just add another if-statements for the new flag? E.g:
 ```suggestion
        {{- if and ( or (eq (toString .Values.cloudsql.enabled) "true") (eq (toString .Values.cloudsql.enabled) "True")) .Values.cloudsql.enabled }}
        {{- if and ( or (eq (toString .Values.cloudsql.useSqlProxy) "true") (eq (toString .Values.cloudsql.useSqlProxy) "True")) .Values.cloudsql.useSqlProxy }}
```
And with an additional "{{- end }}" below.